crm_stage_probability/models/crm_lead.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, fields, models, tools
+
+
+class CrmLead(models.Model):
+
+    _inherit = "crm.lead"
+
+    def _default_probability(self):
+        if "default_stage_id" in self._context:
+            stage_id = self._context.get("default_stage_id")
+        else:
+            stage_id = self._default_stage_id()
+        if stage_id:
+            return self.env["crm.stage"].browse(stage_id).probability
+        return 10
```

Comment:
I will do only one return with one variable like

```python
    def _default_probability(self):
        probability = 10
        if "default_stage_id" in self._context:
            stage_id = self._context.get("default_stage_id")
        else:
            stage_id = self._default_stage_id()
        if stage_id:
            probability = self.env["crm.stage"].browse(stage_id).probability
        return probability
```

Reply:
It's actually a copy-paste from https://github.com/odoo/odoo/blob/12.0/addons/crm/models/crm_lead.py#L56-L63 so :man_shrugging:

Reply:
Can you add Odoo SA as one of the authors of the module, if you copy pasted code from their repo ?

Reply:
(https://www.gnu.org/licenses/gpl-faq.html#AllCompatibility -> in case people are wondering, it is allowed to licence this module under AGPL, even with LGPL code copied)